README.md

```diff
@@ -23,3 +23,24 @@ The Deno language server operates on the following base scopes:
 The server **does not automatically start** for the scopes listed above - it's expected that you enable it for a given project by running `LSP: Enable Language Server in Project` from the _Command Palette_.
 
 You may edit the default settings by running `Preferences: LSP-Deno Settings` from the _Command Palette_.
+
+# Disable LSP-typescript
+
+It is recommended to disable LSP-typescript when working with LSP-deno.
+
+Create a sublime project file with the following contents to disable LSP-typescript per project:
+```json
+{
+  "folders": [{"path": "."}],
+  "settings": {
+    "LSP": {
+      "Deno": {
+        "enabled": true
+      },
+      "LSP-typescript": {
+        "enabled": false
+      }
+    }
+  }
+}
+```
```

Comment:
Maybe integrated it better with the section above that talks about enabling this server using Command Palette. We can similarly mention that LSP-typescript can be disabled using command palette instead of editing project manually.